### PR TITLE
Update SRG-OS-000324-GPOS-00125 for RHEL 9

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:
- Update SRG-OS-000324-GPOS-00125 for RHEL 9
- Add fix tools rules in SRG-OS-000324-GPOS-00125

#### Rationale:

RHEL 9 STIG

### CODE REVIEW

File: controls/stig_rhel9/SRG-OS-000324-GPOS-00125.yml [hunk 1]
```diff
@@ -2,18 +2,15 @@ controls:
...
             - disable_ctrlaltdel_burstaction
             - disable_ctrlaltdel_reboot
             - no_tmux_in_shells
             - service_debug-shell_disabled
-            - securetty_root_login_console_only
             - sysctl_fs_protected_hardlinks
             - sysctl_fs_protected_symlinks
-            - sebool_ssh_sysadm_login
-            - selinux_user_login_roles
             - package_sudo_installed
```
Comment (ggbecker) [inline controls/stig_rhel9/SRG-OS-000324-GPOS-00125.yml:15]: Cross referencing this SRG with the rules listed here I noticed that in RHEL8 STIG for example, only 
```
            - sysctl_fs_protected_hardlinks
            - sysctl_fs_protected_symlinks
```
have the SRG mapped, other rules don't have it. In RHEL7 STIG there are four items that have this SRG, we may want to double check this. We may want to check if rules here are actually applicable to RHEL8/RHEL9 some of them are enabled in RHEL8/9 content but maybe they don't make sense anymore.

Comment (ggbecker) [reply controls/stig_rhel9/SRG-OS-000324-GPOS-00125.yml:15]: this should help us quickly identify rules and their SRG mapping: https://github.com/ComplianceAsCode/content/pull/8493

it wouldn't solve everything though

File: linux_os/guide/system/permissions/files/sysctl_fs_protected_symlinks/rule.yml [hunk 2]
```diff
@@ -31,6 +31,10 @@ references:
 
 {{{ complete_ocil_entry_sysctl_option_value(sysctl="fs.protected_symlinks", value="1") }}}
 
+fix: |-
+    Verify the operating system is configured to enable DAC on hardlinks with the following commands:
```
Comment (jan-cerny) [inline linux_os/guide/system/permissions/files/sysctl_fs_protected_symlinks/rule.yml:35]:
```suggestion
    Verify the operating system is configured to enable DAC on symlinks with the following commands:
```

File: linux_os/guide/system/software/sudo/package_sudo_installed/rule.yml [hunk 3]
```diff
@@ -36,6 +36,9 @@ ocil_clause: 'the package is not installed'
 
 ocil: '{{{ ocil_package(package="sudo") }}}'
 
+fix: |-
+    {{{ describe_package_remove(package="sudo") }}}
```
Comment (jan-cerny) [inline linux_os/guide/system/software/sudo/package_sudo_installed/rule.yml:40]: You want to install sudo, not to remove sudo.

**Review by jan-cerny — CHANGES_REQUESTED** [review]

**Comment by jan-cerny** [comment]
@Mab879 it needs one more rebase :crying_cat_face:

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**